src/Apps/Artist/Routes/AuctionResults/Components/ImageWithFallback.tsx

```diff
@@ -8,3 +9,18 @@ export const ImageWithFallback = ({ Fallback, ...props }) => {
     return <img onError={() => setFallback(true)} {...props} />
   }
 }
+
+export const renderFallbackImage = () => {
+  return (
+    <Box bg="black5" width="100%" height="100%">
+      <Flex
+        alignItems="center"
+        justifyContent="center"
+        width="100%"
+        height="100%"
+      >
+        <NoArtworkIcon width="28px" height="28px" fill="black30" />
```

Comment:
It's worth noting that the icon is probably going to be much smaller than the size of our images here. Might be worth trying to break one of the URLs and see how it looks.

Reply:
The icon is meant to be small, it sits in the middle of a grey box where the image should be. But testing is a good idea regardless.